Workspaces/ProjectWorkspace/ProjectWorkspace.cs

```diff
@@ -144,6 +158,22 @@ public override void Setup()
 			OnBoundsChanged();
 		}
 
+		public object OnSerializeWorkspace()
+		{
+			var preferences = new Preferences();
+			preferences.scaleFactor = m_ProjectUI.assetGridView.scaleFactor;
+			preferences.selectedFolder = m_ProjectUI.folderListView.selectedFolder;
+			return preferences;
+		}
+
+		public void OnDeserializeWorkspace(object obj)
+		{
+			var preferences = (Preferences)obj;
+			m_ProjectUI.assetGridView.scaleFactor = preferences.scaleFactor;
+			m_ProjectUI.folderListView.SelectFolder(preferences.selectedFolder);
```

Comment:
I would recommend that we also save out m_ExpandStates (or at least a list of the guids that are expanded). If a guid changes, we'll lose the expand state, but there shouldn't be any ill effect, as long as we strip serialized expand states that don't have a matching guid at some point.